docs/storage_snippets.py

```diff
@@ -138,7 +139,8 @@ def delete_blob(client, to_delete):
     from google.cloud.exceptions import NotFound
     client = storage.Client()
     bucket = client.get_bucket('my-bucket')
-    assert isinstance(bucket.list_blobs(), list)
+    blobs = list(bucket.list_blobs())
+    assert isinstance(blobs, list)
```

Comment:
The comment below seems to indicate you could even saw `len(blobs) == 1`? Though probably not with a bucket called `my-bucket`?